package/kernel/linux/modules/fs.mk

```diff
@@ -356,13 +386,13 @@ $(eval $(call KernelPackage,fs-nfs-common))
 define KernelPackage/fs-nfs-common-v4
   SUBMENU:=$(FS_MENU)
   TITLE:=Common NFS V4 filesystem modules
+  DEPENDS:=+kmod-fs-nfs-common
   KCONFIG+=\
-	CONFIG_SUNRPC_GSS\
-	CONFIG_NFS_V4=y\
-	CONFIG_NFSD_V4=y
-  DEPENDS:= @BROKEN
+	CONFIG_SUNRPC_GSS
   FILES+=$(LINUX_DIR)/net/sunrpc/auth_gss/auth_rpcgss.ko
-  AUTOLOAD=$(call AutoLoad,30,auth_rpcgss)
+  FILES+=$(LINUX_DIR)/lib/oid_registry.ko
+  AUTOLOAD=$(call AutoLoad,31,auth_rpcgss)
+  AUTOLOAD=$(call AutoLoad,31,oid_registry)
```

Comment:
I think you do not need the AUTOLOAD lines at all and just let it be loaded from the NFS library.

Reply:
You're right that these modules are unnecessary for NFS4 without RPC GSS authentication which lacks userspace support anyway.

This package should be renamed to `kmod-fs-nfs-rpcsec` or something similar and:
  1. the dependency from `kmod-fs-nfs-v4` removed,
  2. module `rpcsec_gss_krb5.ko` should be included.
That would provide separate package with kernel support for `sec=krb5`.